### DISCUSSION TITLE
[Enhancement] [carousel-item] 开放 scale prop 使 el-carousel-item 更容易定制化

### DISCUSSION THREAD

**Discussion by a145789** [element-plus/element-plus#17616] [Feature Request]
Feature type: **`Enhancement`**

## Details

- Enhanced component name: `carousel-item`

## Description

当 `carousel` 为 `card` type 时， `scale` 是固定的 `0.86`，这个倍数无法满足一些 `UI` 的需求，开放 `scale` `prop` 使更加容易定制化

```vue
<el-carousel type="card" :scale="1"></el-carousel>
```

### Additional comments

_(empty)_

**Comment by warmthsea** [comment]
https://github.com/element-plus/element-plus/pull/17621
